website/docs/patterns/form-patterns/partials/guidelines/guidelines.md

```diff
@@ -0,0 +1,236 @@
+
+#### Multiple sections
+
+Breaking up a form into multiple sections based on the relationship between fields and categories of sections can make a long form seem less complex. Introducing typographic elements and dividers can further aid the hierarchy of the form and differentiate sections from one-another.
+
+#### Multiple steps or pages
+
+For exceedingly long, complex forms (e.g., creating a cluster), consider breaking the form into multiple steps or pages. A [Stepper](/components/stepper-indicator) can be used to annotate status and location in a multi-step form.
+
+!!! Info
+
+As a general note, special care should be taken to _reduce_ the number of fields whenever possible.
+
+- Long forms are more challenging for the user which can result in a lower completion rate.
+- If a field is optional, consider whether it should be included in the form at all. If the information isn't crucial to the experience or feature, why is it being collected at all?
+!!!
+
+#### Using dividers
+
+Dividers introduce more visual hierarchy and differentiation in long, complex forms.
+
+Only use dividers between sections of a form, not between fields or other smaller elements within a section. A divider should be used to break up different types of content and categories within a form.
+
+When using a divider, include a 24px gap above and below to separate the divider from surrounding form elements, otherwise it can appear "attached" to a specific section or field.
+
+![Spacing using dividers](/assets/patterns/form-patterns/divider-spacing.png =500x*)
+
+!!! Dont
+
+Dividers should not be used trailing the last section of a form between the fields and the button set or actions. Instead, the section spacing of 32px should be used.
+!!!
+
+## Width
+
+The width of a form and the fields it contains is largely dependent on the context the form is in, but adhering to these high-level guidelines can make a complex form approachable and introduce consistency in the UX.
+
+#### Form width
+
+A form should hug the content and the fields it contains.
+
+![Form width](/assets/patterns/form-patterns/form-width-example.png)
+
+#### Field width
+
+The width of a field within a form should be wide enough to account for the estimated width of the content it accepts. This gives the user an accurate sense of the length and type of the content the field accepts and is important in setting user expectations.
+
+#### Single column layout
+
+If a form uses a single-column layout, consider setting a maximum width on the form; this can generally be best achieved by using a size or unit that is **relative** the page, viewport, or container size. This can commonly be achieved by using a viewport width unit (`vw`), percentage width (e.g., `50%`), or a character unit (`ch`).
```

Comment:
Is there a reason we aren't recommending single-column layouts? It feels like this is an easy opportunity for us to gain alignment across products.

Reply:
It might make sense to recommend this as the default starting point and only to diverge if absolutely necessary. I'll reword some of the content here and reorganize it